CROSSSECTOR/DIH/DigitalInnovationHub/ADOPTERS.yaml

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml

```

Comment:
it would be good to fill it with the actual values where this data model is being used

Reply:
please change the brackets in the first line. ok for the rest